tutorial/quickstart.md

```diff
@@ -1,10 +1,13 @@
 # Pyccel's Quickstart Guide
 
-Pyccel is a **static compiler** for Python 3, using Fortran or C as backend language, with a focus on high-performance computing (HPC) applications.
+Pyccel is a **static compiler** for Python 3, using Fortran or C as a backend language, with a focus on high-performance computing (HPC) applications.
 
-Pyccel's main goal is to accelerate the transition from **prototype** to **production** in scientific computing, where programmers usually develop their prototype code in a user-friendly interactive language like Python, but they later need a statically compiled language like Fortran/C/C++ (as well as SIMD vectorization, parallel multi-threading, MPI parallelization, GPU offloading, etc.) in order to achieve the performance required by their final application.
+Pyccel's main goal is to resolve the main bottleneck in scientific computing, that is the transition from **prototype** to **production**.
+Programmers usually develop their prototype code in a user-friendly interactive language like Python, but their final application requires an HPC implementation and therefore a new production code.
+In most cases this is written in a statically compiled language like Fortran/C/C++, and it uses SIMD vectorization, parallel multi-threading, MPI parallelization, GPU offloading, etc.
 
-Pyccel generates very fast Fortran or C code which is **human-readable**, hence the expert programmer can easily profile the code on the target machine and further optimize it.
+We believe that this expensive process can be avoided, or at least drastically reduced, by using Pyccel to accelerate the most computationally intensive parts of the Python prototype.
+Not only the Pyccel-generated Fortran or C code is very fast, but it is **human-readable**; hence the expert programmer can easily profile the code on the target machine and further optimize it.
```

Comment:
`Not only the Pyccel-generated Fortran or C code is very fast`->`Not only is the Pyccel-generated Fortran or C code very fast`

Reply:
So the verb goes in second place in a sentence?! :open_mouth: Are we writing in German now? :sweat_smile:

Reply:
I've fixed this in commit 47e3382

Reply:
I've no idea why it works like that, but it does when you have "not only" at the start of a sentence
https://dictionary.cambridge.org/grammar/british-grammar/not-only-but-also